packages/router/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@loopback/router",
+  "version": "0.0.1",
+  "description": "Common interfaces and helper functions to build a routing engine for LoopBack 4.",
+  "keywords": [
+    "loopback-extension",
+    "loopback"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=10.16"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run rebuild",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "packages/router"
+  },
+  "author": "IBM Corp.",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "peerDependencies": {
+    "@loopback/core": "^2.13.1"
+  },
+  "dependencies": {
+    "tslib": "^2.0.0"
+  },
+  "devDependencies": {
+    "@loopback/build": "^6.2.8",
+    "@loopback/core": "^2.13.1",
+    "@loopback/testlab": "^3.2.10",
+    "@types/node": "^10.17.48",
+    "typescript": "~4.1.2"
```

Comment:
Please double-check this TypeScript version range is in sync with what we have in other packages. I am not sure if we haven't upgraded TypeScript to a newer version since this PR was opened.